array.c

```diff
@@ -1431,25 +1436,59 @@ rb_ary_shift(VALUE ary)
 
 /*
  *  call-seq:
- *     ary.shift    -> obj or nil
- *     ary.shift(n) -> new_ary
+ *     array.shift -> object or nil
+ *     array.shift(n) -> new_array
+ *
+ *  Removes and returns leading elements of +self+;
+ *  shifts elements down so that the first remaining
+ *  element is at index 0.
+ *
+ *  Argument +n+, if given, must be an
+ *  {Integer-convertible object}[doc/implicit_conversion_rdoc.html#label-Integer-Convertible+Objects]
+ *
+ *  ---
+ *
+ *  When no argument is given,
```

Comment:
Can this fit on a single line. Doesn't it?

Reply:
Done.